stl/inc/deque

```diff
@@ -1418,10 +1419,7 @@ private:
                 _STD fill(_Mid, _Mid + static_cast<difference_type>(_Count),
                     _Tmp._Get_value()); // fill in values
             }
-            _CATCH_ALL
-            _Erase_last_n(_Mysize() - _Oldsize);

```

Comment:
I did not really look, but is that method here more efficient that plain pop_back?

If so should we use it rather than the while loop?

Reply:
```C++
    void _Erase_last_n(size_type _Count) noexcept {
        for (; 0 < _Count; --_Count) {
            pop_back();
        }
    }
```

Maybe only not checking container size each iteration, but not sure if it isn't optimized anyway.